app/javascript/react/components/ReviewForm.js

```diff
@@ -0,0 +1,175 @@
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw error
+        }
+      })
+      .then(response => response.json())
+      .then(body => {
+        setNotice(body.notice)
+        body 
+      })
+      .catch(error => console.error(`Error in fetch: ${error.message}`))
+  };
+
+  return (
+    <div>
+      {getNotice} <br/>
+      <form onSubmit={handleSubmit} id="reviewForm">
+        <h5>Overall</h5>
+        <StarRating
+          size="large"
+          name="overall"
+          rating={overallRating}
+          hoverRating={overallHoverRating}
+          onStarClick={(value) => {
+            setOverallRating(value)
+          }}
+          value={reviewData.overall}
+          onStarHover={(value) => {
+            setOverallHoverRating(value);
+          }}
+          onMouseLeave={() => {
+            setOverallHoverRating(undefined);
+          }}
+        />
+
+        <h5>Sweetness</h5>
+        <StarRating
+          size="large"
+          name="overall"
+          rating={sweetnessRating}
+          hoverRating={sweetnessHoverRating}
+          onStarClick={(value) => {
+            setSweetnessRating(value)
+          }}
+          onStarHover={(value) => {
+            setSweetnessHoverRating(value);
+          }}
+          onMouseLeave={() => {
+            setSweetnessHoverRating(undefined);
+          }}
+        />
+
+        <h5>Mouth Feel</h5>
+        <StarRating
+          size="large"
+          name="overall"
+          rating={mouthFeelRating}
+          hoverRating={mouthFeelHoverRating}
+          onStarClick={(value) => {
+            setMouthFeelRating(value);
+          }}
+          onStarHover={(value) => {
+            setMouthFeelHoverRating(value);
+          }}
+          onMouseLeave={() => {
+            setMouthFeelHoverRating(undefined);
+          }}
+        />
+
+        <h5>Taste</h5>
+        <StarRating
+          size="large"
+          name="overall"
+          rating={tasteRating}
+          hoverRating={tasteHoverRating}
+          onStarClick={(value) => {
+            setTasteRating(value);
+          }}
+          onStarHover={(value) => {
+            setTasteHoverRating(value);
+          }}
+          onMouseLeave={() => {
+            setTasteHoverRating(undefined);
+          }}
+        />
+
+        <label>
+          Comment (optional):
+          <textarea
+            name="comment"
+            onChange={handleTextInputChange}
+            value={reviewData.comment}
+          />
+        </label>
+
+        <br />
+
+        <label>
+          Brand (required):
+          <textarea
+            name="manufacturer"
+            onChange={handleTextInputChange}
+            value={reviewData.manufacturer}
+          />
+        </label>
+
+        <button type="submit" value="Submit" form="reviewForm">Submit Form</button>
+          <br />
+          <br />
```

Comment:
Yall really need to remove all of the <br> tags and use a smidge of styling instead. If you do EVERYTHING else I will allow this PR to merge, but I would just get it over with and use CSS/Foundation to add the space you need

Reply:
We removed these line breaks.